### PR TITLE
Integration: Ensure no rejections in PL isolation test

### PR DESCRIPTION
**Pull request by yue9944882**
```release-note
NONE
```
/sig api-machinery

### CODE REVIEW

File: test/integration/apiserver/flowcontrol/concurrency_test.go [hunk 1]
```diff
@@ -122,13 +123,20 @@ func TestPriorityLevelIsolation(t *testing.T) {
 
 	time.Sleep(time.Second * 10) // running in background for a while
 
-	reqCounts, err := getRequestCountOfPriorityLevel(loopbackClient)
+	allDispathedReqCounts, rejectedReqCounts, err := getRequestCountOfPriorityLevel(loopbackClient)
```
Comment (logicalhan) [inline test/integration/apiserver/flowcontrol/concurrency_test.go:126]: dispatch is misspelled here.

**Review by logicalhan — APPROVED** [review]
/lgtm

minor misspelling though

**Comment by MikeSpreitzer** [comment]
/cc @MikeSpreitzer 
I will LGTM this again if the spelling is fixed.

File: test/integration/apiserver/flowcontrol/concurrency_test.go (hunk 1 above)
Comment (yue9944882) [reply test/integration/apiserver/flowcontrol/concurrency_test.go]: 👀 nice catch

**Review by MikeSpreitzer — APPROVED** [review]
/LGTM

**Pull request merged**